functions.sh

```diff
@@ -85,4 +85,24 @@ load_settings_file() {
     fi
 }
 
+# in_array - check if an element can be found in an array
```

Comment:
It doesn't seem right to define a function like this which does such a generic thing - shouldn't it come from a library or something?  Or is this indicating that bash isn't the right choice of language for this application?